lib/tapioca/loader.rb

```diff
@@ -24,6 +24,21 @@ def load_bundle(initialize_file, require_file)
       load_rails_engines
     end
 
+    sig { params(environment_load: T::Boolean, eager_load: T::Boolean).void }
+    def load_rails(environment_load: false, eager_load: false)
+      return unless File.exist?("config/application.rb")
+
+      safe_require("rails")
+
+      silence_deprecations
+
+      safe_require("rails/generators/test_case")
+      safe_require("./config/application")
```

Comment:
```suggestion
      safe_require("./config/environment")
```

if environment_load instead of `application`.